BIPs/2023-W27/BIP-350.report.txt

```diff
@@ -0,0 +1,11 @@
+File name: BIPs/2023-W27/BIP-350.json
+COMMIT: `eaf9aabb791c20a82202963d8190aa4d807cbc96`
+CHAIN(S): `mainnet`
+```
++----------+--------------+----------------+----------+--------------------------------------------+--------------------------------------------+-------------------------+---------+-------------------+
+| function | token_symbol | recipient_name |  amount  |               token_address                |             recipient_address              |        raw_amount       |   bip   | transaction_index |
++----------+--------------+----------------+----------+--------------------------------------------+--------------------------------------------+-------------------------+---------+-------------------+
```

Comment:
<img width="757" alt="image" src="https://github.com/BalancerMaxis/multisig-ops/assets/11885129/e9d07a18-baef-4a4c-b717-edf4fa37ef3d">

Amounts check and recipient resolves.